sorts/radix_sort.py

```diff
@@ -1,26 +1,36 @@
-def radix_sort(lst):
+from typing import List
+
+
+def radix_sort(list_of_ints: List[int]) -> List[int]:
+    """
+    radix_sort(range(15)) == sorted(range(15))
```

Comment:
I wonder if doctests like:
```
>>> radix_sort([5, 4, 3, 2, 1])
[1, 2, 3, 4, 5]
```
would be more readable?

Reply:
Perhaps but they would not test two digit values.